client/src/modules/shopping-list/model/reducer.js

```diff
@@ -80,6 +82,8 @@ const lists = (state = {}, action) => {
         [action.payload.listId]: action.payload.data
       };
     }
+    case CohortActionTypes.ARCHIVE_SUCCESS:
```

Comment:
Cohort case in list reducer??

Reply:
When we archive cohort, we want to remove cohort's list from redux also. So list reducer is listening for archive cohort action. In my first approach there was a action creator removeLists in list actions file and it was dispatching in archive cohort action. There was much more unnecessary code. This Marek's solution is much more cleaner.